src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -27,147 +28,86 @@ const styles = {
-  return {
-    data: darCode,
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const submissionDateCellData = ({createDate, darCollectionId, style = {}, label = 'submission-date'}) => {
-  return {
-    data: isNil(createDate) ? '- - ' : formatDate(createDate),
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const statusCellData = ({status = '- -', darCollectionId, style = {}, label = 'status'}) => {
-  return {
-    data: status,
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const CancelCollectionButton = (props) => {
-  const { collection } = props;
-  return h(SimpleButton, {
-    keyProp: `cancel-collection-${collection.id}`,
-    label: 'Cancel',
-    disabled: includes(determineCollectionStatus(collection))(nonCancellableCollectionStatuses),
-    baseColor: Theme.palette.secondary,
-    additionalStyle: {
-      width: '30%',
-      padding: '2%',
-      fontSize: '1.45rem',
-    },
-    onClick: () => props.showConfirmationModal(collection)
-  });
-};
-
-const ResubmitCollectionButton = (props) => {
-  const { collection } = props;
-  return h(SimpleButton, {
-    keyProp: `resubmit-collection-${collection.id}`,
-    label: 'Revise',
-    baseColor: Theme.palette.secondary,
-    additionalStyle: {
-      width: '30%',
-      padding: '2%',
-      fontSize: '1.45rem',
-    },
-    onClick: () => props.showConfirmationModal(collection)
-  });
-};
-
-const actionsCellData = ({collection, showConfirmationModal}) => {
-  const { darCollectionId } = collection;
-  const cancel = {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'cancel-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
-        },
-        key: `cancel-collection-cell-${darCollectionId}`
-      },
-      [h(CancelCollectionButton, {collection, showConfirmationModal})]
-    )};
-  const revise = {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'resubmit-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
-        },
-        key: `resubmit-collection-cell-${darCollectionId}`
-      },
-      [h(ResubmitCollectionButton, {collection, showConfirmationModal})]
-    )
-  };
-
-  return isCollectionCanceled(collection) ? revise : cancel;
+  const {darCode, name, submissionDate, pi, institution, datasetCount, status, actions} = columnHeaderFormat;
+  return [darCode, name, submissionDate, pi, institution, datasetCount, status, actions];
 };
 
 
-const processCollectionRowData = (collections, showConfirmationModal) => {
+const processCollectionRowData = (collections, showConfirmationModal, actionsFlag) => {
   if(!isNil(collections)) {
     return collections.map((collection) => {
-      const { darCollectionId, darCode, createDate } = collection;
+      const { darCollectionId, darCode, createDate, datasets } = collection;
+      const institution = collection.createUser.institution.name;
```

Comment:
Another interesting thing about Users: they may not belong to an institution. When users first join, they are not given an institution since we generate a user account off of their gmail auth log in. The user has to update that value themselves via the User Profile Page.

That said, with or without an institution, a user can still submit DAR drafts and DARs, so doing `collection.createUser.institution.name` might return an error. You'll have to check if institution exists, and if it does, get the name. Otherwise settle on a default value like an empty string or "- -"